tests/main/snap-user-service-socket-activation/task.yaml

```diff
@@ -15,6 +15,7 @@ prepare: |
     tests.session kill-leaked
 
 restore: |
+    snap remove test-snapd-user-service-sockets --purge
```

Comment:
Maybe a small comment here that says that this needs to happen before the tests.session line below? So that this won't get changed accidental the future?

Reply:
it would be ideal to defer removal of this via tests.cleanup, and add a comment there IMHO

Reply:
Thanks for the review, the comment is added